frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/PeriodsAgoMenuOption.tsx

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useRef } from "react";
+import type { KeyboardEvent, MouseEvent } from "react";
+import { Group, Text, Box } from "metabase/ui";
+import type { SmartScalarComparisonPeriodsAgo } from "metabase-types/api";
+import type { COMPARISON_TYPES } from "metabase/visualizations/visualizations/SmartScalar/constants";
+import { NumberInputStyled } from "./PeriodsAgoMenuOption.styled";
+import { MenuItemStyled } from "./MenuItem.styled";
+import type { HandleEditedValueChangeType } from "./SmartScalarSettingsWidgets";
+
+type PeriodsAgoMenuOptionProps = {
+  "aria-selected": boolean;
+  editedValue?: SmartScalarComparisonPeriodsAgo;
+  maxValue: number;
+  name: string;
+  handleChange: HandleEditedValueChangeType;
+  type: typeof COMPARISON_TYPES.PERIODS_AGO;
+};
+
+const MIN_VALUE = 2;
+
+export function PeriodsAgoMenuOption({
```

Comment:
Updated component name according to https://github.com/metabase/metabase/pull/36666#discussion_r1432645994